deploy/operator/after_upgrade.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+export UPGRADE_OPERATOR="false"
```

Comment:
I'm pretty bad at naming, but I'm wondering if this is too generic. Maybe `IS_OPERATOR_UPGRADE_FLOW` or `UPGRADE_ASSISTED_OPERATOR`? Since this is used to denote select which OCP version to start with.

Just trying to think about reading this. 🤔

Reply:
`UPGRADE_ASSISTED_OPERATOR` sounds good.